taxi/templates/taxi/index.html

```diff
@@ -0,0 +1,9 @@
+{% extends "base.html"%}
+
+{% block content%}

```

Comment:
Consider adding a space after `{% block content %}` to maintain consistency with other block tags.